core/CMakeLists.txt

```diff
@@ -39,10 +39,10 @@ detray_add_library( detray_core core
    "include/detray/definitions/detail/accessor.hpp"
    "include/detray/definitions/track_parametrization.hpp"
    # geometry include(s)
+   "include/detray/geometry/detector_volume.hpp"
```

Comment:
You also need to remove `viewable_collection.hpp`

Reply:
Oh, I was sure I had removed it. It is gone in the next PR, I promise